mods/map/init.lua

```diff
@@ -0,0 +1,84 @@
+-- Mod global namespace
+
+map = {}
+
+
+-- Cache creative mode setting
+
+local creative_mode_cache = minetest.settings:get_bool("creative_mode")
+
+
+-- Global function to allow overriding
+
+function map.enable(player)
```

Comment:
Does not match with what it actually does. How about `map.update` ?

Reply:
map.update_hud_flags ?

Reply:
^ Ok too. Either function name sounds good.

Reply:
Ok.